whirlpool/eventsocket.py

```diff
@@ -84,10 +116,12 @@ async def _run(self):
 
             self._websocket = None
 
-        if self._running and self._reconnect_tries > 0:
```

Comment:
I don't think we should remove this.
We can retry 3 times fast, but then we should have a bigger wait time (1hour) after that.
If the server goes down, we don't want HA to start hammering it with a lot of requests while it is trying to boot up, and risk bringing it down again with a DOS :smile:

Reply:
Find a try at this - fast retry for 3 attempts then slow down forever....

Reply:
On this note - they do seem to send the 1001 message randomly - sometimes multiple times quickly, other times once / hour, and I have seen it go to auth key expiration (aka no 1001's for 2 or 3 days except for key renewal)   I'll get this change running in my local HA and watch for any odd behavior.